### PR TITLE
Small fixes in the proposal process for cEUR/Release 3

### PR DESCRIPTION
**Pull request by nambrot**
### Description

This PR cherry-picks @AlexBHarley 's changes to accomodate CR3 with cEUR related changes

### CODE REVIEW

**Review by AlexBHarley — COMMENTED** [review]
Seems like bad practice to approve my own cherry picked code, but lgtm!

**Review by AlexBHarley — APPROVED** [review]

File: packages/sdk/contractkit/src/proxy.ts [hunk 1]
```diff
@@ -116,6 +117,7 @@ const initializeAbiMap = {
   ReserveProxy: findInitializeAbi(ReserveABI),
   SortedOraclesProxy: findInitializeAbi(SortedOraclesABI),
   StableTokenProxy: findInitializeAbi(StableTokenABI),
+  StableTokenEURProxy: findInitializeAbi(StableTokenABI),
```
Comment (yorhodes) [inline packages/sdk/contractkit/src/proxy.ts:120]: good catch

File: packages/cli/src/commands/governance/propose.ts [hunk 2]
```diff
@@ -52,8 +52,7 @@ export default class Propose extends BaseCommand {
     // builder.addProxyRepointingTx
 
     const proposal = await builder.build()
-
-    printValueMapRecursive(proposalToJSON(this.kit, proposal))
+    printValueMapRecursive(await proposalToJSON(this.kit, proposal))
```
Comment (yorhodes) [inline packages/cli/src/commands/governance/propose.ts:55]: did this just never work?

**Review by yorhodes — APPROVED** [review]

File: packages/cli/src/commands/governance/propose.ts (hunk 2 above)
Comment (AlexBHarley) [reply packages/cli/src/commands/governance/propose.ts:55]: Yeah believe so

**Pull request merged**